persistence/src/main/java/edu/unc/lib/dl/persist/services/destroy/DestroyObjectsJob.java

```diff
@@ -142,13 +155,21 @@ public void run() {
 
     private void destroyTree(RepositoryObject rootOfTree) throws FedoraException, IOException,
             FcrepoOperationFailedException {
+        // Add the root of the tree to delete
+        deletedObjCount += 1;
+        deletedObjIds.add(rootOfTree.getPid().getUUID());
+
         if (rootOfTree instanceof ContentContainerObject) {
             ContentContainerObject container = (ContentContainerObject) rootOfTree;
             List<ContentObject> members = container.getMembers();
             for (ContentObject member : members) {
+                deletedObjIds.add(member.getPid().getUUID());
```

Comment:
Could you update DestroyObjectsJobIT to check for an event being added to the parent? You should be able to check that the right number of objects are destroyed at least